api_compatibility_policy.md

```diff
@@ -41,12 +41,20 @@ alpha or beta:
 that all `CustomRun` controllers are required to support is considered a [backwards incompatible change](#backwards-incompatible-changes),
 and follows the [beta policy](#beta-crds) for backwards incompatible changes.
 
-`v1beta1.ClusterTask` is a deprecated beta CRD. New features will not be added to `ClusterTask`.
-
 `v1beta1.ResolutionRequest` is a beta CRD. Adding new fields to `ResolutionRequest`
 that all resolvers are required to support is considered a [backwards incompatible change](#backwards-incompatible-changes),
 and follows the [beta policy](#beta-crds) for backwards incompatible changes.
 
+The following CRDs are deprecated. New features will not be added, but they will receive bug fixes.
```

Comment:
Can we please add a timeline of when this deprecation was announced and until when these API will receive bug fixes? It will help the users to understand and plan their upgrade. Thanks!

Reply:
Is there info missing from docs/deprecations.md that would be helpful here?

Reply:
There is no missing info from `docs/deprecations.md` but at the same time there is no harm in adding a bit more details here for the users. As a user, how would I know when these changes were introduced if I am just looking at the [api_compatibility_policy.md#crd-api-versions](https://github.com/tektoncd/pipeline/blob/86ff3643242481624eed6dd35ad95b63d1a6a01e/api_compatibility_policy.md#crd-api-versions). There is no mention of the `docs/deprecations.md`. At the least if we add a link to the deprecations.md here, it will be very helpful for the users.

Reply:
added a link! thanks for the feedback.